src/__configuration__/markdown/copy.ts

```diff
@@ -0,0 +1,24 @@
+// https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript
```

Comment:
does this file belong in the config folder, or is it more of a utility or lib?

Reply:
Moved to shared/utilities.tsx